scripts/generate-merge-message.js

```diff
@@ -57,10 +57,18 @@ function getReleaseSchedule() {
   const firstReleaseYear = 2022;
   const firstReleaseMonth = 2;
```

Comment:
perhaps we could shift this 😄

```suggestion
  const firstReleaseMonth = 1;
```

Reply:
Fair enough! Although that kind of "rewriting history" makes me a bit twitchy :joy:

If we're aiming for pragmatism, would it work to just look at the most recent non-RC release and find the next "Third Tuesday of the month" and avoid anchoring to any sort of start date or assumptions about release schedule?